gallery/visualtests/nxToggle.js

```diff
@@ -25,6 +25,7 @@ describe('NxToggle', function() {
 
       await targetElement.scrollIntoView({ block: 'center' });
       await targetElement.click();
+      await targetElement.moveTo({ xOffset: -10, yOffset: -10 });
```

Comment:
Fixes issue @dturner32 noticed where hover style was present in this applitools check